### PR TITLE
fix: better long message

### PR DESCRIPTION
**Pull request by MalotteAtKivra**
[Jira](https://kivra.atlassian.net/browse/APP-1755)

### CODE REVIEW

File: api-errors.json [hunk 1]
```diff
@@ -253,7 +253,7 @@
     },
     "40109": {
         "short_message": "Sendrequest Or Share Already Exists",
-        "long_message": "An already existing Sendrequest exists or is blocked"
+        "long_message": "An Sendrequest or a Share already exists"
```
Comment (kivra-joarod) [inline api-errors.json:256]: Maybe the sentence should start with 'A' not 'An'?

**Review by kivra-joarod — APPROVED** [review]

File: api-errors.json (hunk 1 above)
Comment (RSBRK) [reply api-errors.json:256]:
```suggestion
        "long_message": "A Sendrequest or Share already exists"
```

Comment (RSBRK) [reply api-errors.json:256]: This is very similar to the short message. Is there any other information we can add to the long message that is not in the short one?

Comment (MalotteAtKivra) [reply api-errors.json]: It's sort of messy since a sendrequest can be used for several different purposes. But I can try to add something.

**Review by RSBRK — APPROVED** [review]

**Pull request merged**